.github/workflows/ci-cd.yml

```diff
@@ -79,6 +79,7 @@ jobs:
         slotscheck -v -m aiohttp
     - name: Install libenchant
       run: |
+        sudo apt-get update
         sudo apt install libenchant-2-dev
```

Comment:
Can we just delete this step my other CIs work well without installing libenchant explicitly. It used to be necessary in Travis CI but the GH images just have some version pre-installed and we don't really have to update it.

Reply:
I'll come back and delete it after I get https://github.com/aio-libs/aiohttp/pull/11054 released

Reply:
https://github.com/aio-libs/aiohttp/pull/11064